examples/ex4/CMakeLists.txt

```diff
@@ -2,36 +2,51 @@
 #
 cmake_minimum_required(VERSION 3.14)
 
+find_package(dlite REQUIRED)
+
 project(dlite-example-4
   VERSION 0.1
   LANGUAGES Fortran C)
 
+add_compile_options("-DDLITE_ROOT=${DLITE_ROOT}")
+#TODO: make this platform independent, need to add support for long strings
+set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -ffree-line-length-none")
+
 list(APPEND CMAKE_MODULE_PATH
   ${DLITE_ROOT}/cmake
   ${DLITE_ROOT}/cmake/Modules
   ${DLITE_ROOT}/share/cmake
   ${DLITE_ROOT}/share/cmake/Modules
   )
-include(Finddlite)
-find_package(dlite)
 
+
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${DLITE_ROOT}/share/dlite/cmake)
```

Comment:
This can be included in the `list(APPEND ...)` above

Reply:
Have no clue why, but that does not work

Reply:
Ok, then we keep it